src/Type/Definition/InputObjectType.php

```diff
@@ -142,6 +143,24 @@ protected function initializeField($nameOrIndex, $field): void
         $this->fields[$field->name] = $field;
     }
 
+    /**
+     * Parses an externally provided value (query variable) to use as an input.
+     *
+     * Should throw an exception with a client friendly message on invalid values, @see ClientAware.
+     *
+     * @param mixed $value
```

Comment:
```suggestion
     * @param array<string, mixed> $value
```

I don't think it can ever be anything else.

Reply:
You're right! Changed it.